src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -3280,5 +3269,91 @@ static void F()
 
             ExitBreakState();
         }
+
+        /// <summary>
+        /// Scenario:
+        /// - F5
+        /// - edit and apply edit that deletes non-leaf active statement
+        /// - break
+        /// </summary>
+        [Fact, WorkItem(52100, "https://github.com/dotnet/roslyn/issues/52100")]
```

Comment:
This test is unrelated to the rest of the change but not really worth separate PR :)